docs/Tutorials/CCE.md

```diff
@@ -292,6 +292,9 @@ Here are some notes about the different options in the YAML input file:
   simulation that provided the boundary dataset.
 - `FixSpecNormalization` should always be `False` unless you are using a
   particualy old version of SpEC
+- `DescendingM` should always be `False`. This option is to maintain
```

Comment:
This sentence could be confusing because `DescendingM` depends on how people provide their metric modal data. It can be `True`. I guess you are referring to the output Bondi modal data, which always have ascending m.

Reply:
Yes technically it can be true for metric modal data, but I think we want to encourage people to always have their data in ascending m order (unless it's from spec).